src/Scouting.elm

```diff
@@ -1,65 +1,121 @@
-import Html.Attributes exposing (placeholder, value)
-import Html exposing (input , text)
-import Html.Events exposing (onClick , onInput)
+import Html exposing (input, text)
+import Html.Attributes exposing (placeholder, style, value)
+import Html.Events exposing (onClick, onInput)
+import Maybe.Extra exposing (unwrap)
 
 
-inputwriter : String -> (String -> Msg) -> String -> Html.Html Msg
-inputwriter placeholderInput msg model =
-    Html.div [] [
-    input [placeholder placeholderInput, onInput msg, value model] []
-    ,text "\n"  
-    ]
+inputWriter : String -> (String -> Msg) -> String -> Html.Html Msg
+inputWriter placeholderInput msg model =
+    Html.div [ style "margin-bottom" "1%" ]
+        [ input [ placeholder placeholderInput, onInput msg, value model ]
+            []
+        ]
+
+
+unwrapToString : Maybe Int -> String
+unwrapToString maybeInt =
+    unwrap "" String.fromInt maybeInt
+
+
+type Pressed
+    = Pressed
+    | NotPressed
+    | InputError
+
 
-type Msg =
-    ScoutersNameInput String
+inputChecker : Model -> Model
+inputChecker model =
+    if model.scoutersName == "" || model.scoutedTeamNumber == Nothing || model.scoutedTeamDriverStationPosition == "" || model.matchNumber == Nothing then
+        { model | isPressed = InputError }
+
+    else
+        { model | isPressed = Pressed }
+
+
+type Msg
+    = ScoutersNameInput String
     | TeamNumberInput String
     | MatchNumberInput String
     | DriverStationInput String
-    | Page Bool
+    | Page Pressed
 
 
 type alias Model =
-    {scoutersName : String
-    , scoutedTeamNumber : String
-    , matchNumber : String
+    { scoutersName : String
+    , scoutedTeamNumber : Maybe Int
+    , matchNumber : Maybe Int
     , scoutedTeamDriverStationPosition : String
-    , isPage : Bool
+    , isPressed : Pressed
     }
 
+
 main : Program () Model Msg
-main = 
-    Browser.element{init = always (init, Cmd.none), update = update, view = view, subscriptions = always Sub.none }
+main =
+    Browser.element
+        { init = always ( init, Cmd.none )
+        , update = update
+        , view = view
+        , subscriptions = always Sub.none
+        }
+
 
 init : Model
-init = {scoutersName = "", scoutedTeamNumber = "", matchNumber = "", scoutedTeamDriverStationPosition = "", isPage = False}
-
-update : Msg -> Model -> (Model, Cmd Msg)
-update msg model = 
-    case msg of 
-        ScoutersNameInput string ->
-            ({model | scoutersName = string }, Cmd.none)
-        TeamNumberInput string ->
-            ({model | scoutedTeamNumber = string }, Cmd.none)
-        MatchNumberInput string ->
-            ({model | matchNumber = string }, Cmd.none)
-        DriverStationInput string ->
-            ({model | scoutedTeamDriverStationPosition = string }, Cmd.none)
+init =
+    { scoutersName = "", scoutedTeamNumber = Nothing, matchNumber = Nothing, scoutedTeamDriverStationPosition = "", isPressed = NotPressed }
+
+
+update : Msg -> Model -> ( Model, Cmd Msg )
```

Comment:
Notice how you always return `Cmd.none`?
It means you should return a `Cmd Msg` 🙂